impeller/display_list/dl_dispatcher.cc

```diff
@@ -1104,9 +1104,23 @@ void DlDispatcher::drawDisplayList(
 void DlDispatcher::drawTextBlob(const sk_sp<SkTextBlob> blob,
                                 SkScalar x,
                                 SkScalar y) {
-  canvas_.DrawTextFrame(TextFrameFromTextBlob(blob),  //
-                        impeller::Point{x, y},        //
-                        paint_                        //
+  const auto text_frame = TextFrameFromTextBlob(blob);
+  if (paint_.style == Paint::Style::kStroke) {
+    auto path = skia_conversions::PathDataFromTextBlob(blob);
+    auto bounds = text_frame.GetBounds();
+    if (!bounds.has_value()) {
+      return;
+    }
+    canvas_.Save();
+    canvas_.Translate({x + bounds->origin.x, y + bounds->origin.y, 0.0});
```

Comment:
Is the frame bounds origin actually the right thing to be adding here? It would seem weird for Skia's API to circularly depend on the text frame bounds origin when coming up with path coords to output. Needing to add in something like the baseline offset would make sense, though.

Reply:
To be honest I don't really know. This is similar to the approach I used when drawing with different color sources on top of the text: https://github.com/flutter/engine/blob/main/impeller/aiks/canvas.cc#L529-L534